modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/js/StyleBookManagementToolbarPropsTransformer.js

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {openSimpleInputModal} from 'frontend-js-web';
+
+export default function propsTransformer({
+	additionalProps: {copyStyleBookEntryURL, exportStyleBookEntriesURL},
+	portletNamespace,
+	...otherProps
+}) {
+	const copySelectedStyleBookEntries = () => {
+		const form = document.getElementById(`${portletNamespace}fm`);
+
+		const styleBookEntryIds = document.getElementById(
+			`${portletNamespace}styleBookEntryIds`
+		);
+
+		if (!form && !styleBookEntryIds) {
+			return;
+		}
+
+		styleBookEntryIds.setAttribute(
+			'value',
+			Liferay.Util.listCheckedExcept(form, `${portletNamespace}allRowIds`)
+		);
+
+		const styleBookEntryFm = document.getElementById(
+			`${portletNamespace}styleBookEntryFm`
+		);
+
+		if (!styleBookEntryFm) {
+			return;
+		}
+
+		submitForm(styleBookEntryFm, copyStyleBookEntryURL);
```

Comment:
SF nit. If it is just one line in `if` block, justification for early return is poor. The goal of the convention is to have code more readable. For example:
```js
if (condition) {
    // 100 lines of code

    // some complex logic
}
```
is not great, because we need to have the check in mind. In practice, we often have many layers of nesting. On the other hand, if we have

```js
if (!condition) {
   return;
}
// we don't care about condition any more 

// 100 lines of code, with lesser indentation

// some complex logic
```
it's easier to focus on complex logic.

But this is kinda pointless if the block of code is simple and small.

```js
if (styleBookEntryFm) {
   submitForm(styleBookEntryFm, copyStyleBookEntryURL);
}
```
is probably more readable than the alternative. This is subjective, so either is not a blocker.

Reply:
I agree with you on that @markocikos.  I was just following the convention that we had used in all the previous management toolbar migrations that have already went in.  Will make the change here, but will also review what has already gone in and make the changes there as well.  Thanks for the feedback!